src/AppContainer.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useMemo } from 'react';
+import {
+  Navigate,
+  Route,
+  Routes,
+  useLocation,
+} from 'react-router-dom';
+import { App } from './App';
+import { FilterStatus } from './types/FilterStatus';
+import { useTodo } from './TodoContext';
+
+export const AppContainer: React.FC = () => {
+  const {
+    todos,
+  } = useTodo();
+  const { pathname } = useLocation();
+  const [filter, setFilter] = useState(FilterStatus.all);
+
+  useMemo(() => {
+    if (pathname === '/completed') {
+      setFilter(FilterStatus.completed);
+    }
+
+    if (pathname === '/active') {
+      setFilter(FilterStatus.active);
+    }
+  }, [pathname]);
```

Comment:
Not fixed. + uncoorect usign useMemo
```suggestion

```
![2023-09-14_16h01_41](https://github.com/mate-academy/react_todo-app/assets/61904995/4a227e03-c392-4d65-9c73-77c28cdc3a88)